README.md

```diff
@@ -8,92 +8,5 @@ Can be used in place of Ruby's Logger
 
 In fact, it (currently) returns an instance of Logger.
 
-Forked from [loggiler](https://github.com/freeformz/logglier).
+Forked from (<https://github.com/zshannon/cloudwatchlogger>).
 
-Usage
```

Comment:
@manikandan0603 - why did you delete the readme bro? 😓

Reply:
Since we are gonna use notion doc for implementation, I removed the old content.  What do you suggest we do here?

Reply:
I suggest add the stuff from Notion here? It should be possible to use this repo on its own, if anyone else also wants to use it, since the gem is public

Reply:
@mikevic - Updated the Readme.